tests/utils/log/test_s3_task_handler.py

```diff
@@ -136,7 +138,8 @@ def test_read_raises_return_error(self):
 
     def test_write(self):
         with mock.patch.object(self.s3_task_handler.log, 'error') as mock_error:
-            self.s3_task_handler.s3_write('text', self.remote_log_location)
+            successful_write = self.s3_task_handler.s3_write('text', self.remote_log_location)
+            assert successful_write is True
```

Comment:
```suggestion
            self.assertTrue(successful_write)
```

That way if it isn't we get better error diagnostics.